cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -16,6 +16,8 @@
 // under the License.
 
 #include "arrow/filesystem/azurefs.h"
+#include <boost/process.hpp>
```

Comment:
Could you use https://github.com/apache/arrow/blob/7cbbd3ee95bc0f7d20bb358e2978e2eb18a05304/cpp/src/arrow/filesystem/gcsfs_test.cc#L18-L35 ?

Reply:
I don't see a reason not to